.github/workflows/codeql-analysis.yml

```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - 'terraform/**'
```

Comment:
humm that will not run and the check that is required will not be valid and will block

we should use the `dorny/paths-filter@v2` to filter then that will properly skip the job

Reply:
hmm, its worked across other workflows in this repo....... I guess I don't understand the difference?

Reply:
Oh really? That is good news then GitHub changed that behavior nice